listRepos.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
This should probably be `/bin/bash`.  On Ubuntu, `/bin/sh` is, unfortunately, no longer bash (Ubuntu decided to switch it to another shell called "dash").